### PR TITLE
Aaaaaaaand some more

### PR DESCRIPTION
**Pull request by dougaitken**
I'd say leave it open but I edited readme.md by making "Everywhere" into "Worldwide" :earth_africa: :earth_americas: :earth_asia:  I might cause the odd merge conflict :hammer:  whoops :sunglasses:  Because I like breaking things but mainly because that's what is mentioned in CONTRIBUTING.md :+1:

### CODE REVIEW

**Comment by jessicard** [comment]
Ahh wonderful!! Thanks for all of your work on this @dougaitken. I'll keep it open until you say otherwise :) Hopefully we won't run into a nasty conflict.

File: README.md [hunk 1]
```diff
@@ -46,90 +47,91 @@ Cards Against Humanity | https://cardsagainsthumanity.com/ |
...
 [Casumo](/company-profiles/casumo.md) | https://www.casumo.com/ | Europe
 Catalyze | https://catalyze.io/ |
+[Chargify](/company-profiles/chargify.md) | https://www.chargify.com/ | USA
 charity: water | http://www.charitywater.org/ |
 Chef | https://www.chef.io/ |
 CircleCI | https://circleci.com/ |
-[Citrusbyte](/company-profiles/citrusbyte.md) | https://citrusbyte.com/ | Everywhere
-Clevertech | http://www.clevertech.biz/ |
+[Citrusbyte](/company-profiles/citrusbyte.md) | https://citrusbyte.com/ | Worldwide
+[Clevertech](/company-profiles/clevertech.md) | http://www.clevertech.biz/ | Worldwide
```
Comment (dougaitken) [inline README.md]: This is the company I tweeted about!

**Comment by dougaitken** [comment]
You're welcome, I'm just glad I can do this. I know it's not amazing open source work but it's getting me involved until I can somehow create a 2nd life to give me time to get "experience" to be employable for one of these companies :+1: 

Best not risk it with any more breaks, #shipit :shipit: for now :octocat:

**Comment by jessicard** [comment]
Woohoo, thanks!! :bouquet:

**Pull request merged**